pkg/storage/mongo/all_test.go

```diff
@@ -0,0 +1,43 @@
+package mongo
+
+import (
+	"testing"
+
+	"github.com/gomods/athens/pkg/storage"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	baseURL = "testbaseurl"
+	module  = "testmodule"
+	version = "v1.0.0"
+)
+
+var (
+	// TODO: put these values inside of the suite, and generate longer values.
+	// This should help catch edge cases, like https://github.com/gomods/athens/issues/38
+	//
+	// Also, consider doing something similar to what testing/quick does
+	// with the Generator interface (https://godoc.org/testing/quick#Generator).
+	// The rough, simplified idea would be to run a single test case multiple
+	// times over different (increasing) values.
+	mod = []byte("123")
+	zip = []byte("456")
+)
+
+type MongoTests struct {
+	suite.Suite
+	storage storage.StorageConnector
+}
+
+func (d *MongoTests) SetupTest() {
+	store := NewMongoStorage("mongodb://127.0.0.1:27017")
```

Comment:
Is this worth configuring, or is there a presumption that mongo will be available everywhere that wants to run unit tests?
Perhaps guard with a build flag, if that's the case?

Reply:
also understand the irony of this comment, given it was probably copy/paste from my mongo impl :)

Reply:
This is not a blocking change - so consider this a LGTM.

Reply:
fwiw, I'm ok with keeping this for now since we have a docker-compose file in place to run mongo locally - we should consider adding a build flag when our corpus of tests grows